examples/quickstart/jupyter-notebooks/README.md

```diff
@@ -65,6 +65,13 @@ Make sure you meet the following requirements before starting the Jupyter-based
     jupyter notebook --port 3001
     ```
 
+- The Python API client for Druid. From the Druid repo in `examples/quickstart/jupyter-notebooks/druidapi`
```

Comment:
Do we say anywhere that they need to clone the repo first in this file? It says it in the druidapi README

Also `From the Druid repo in ...` sounds kinda awkward although I'm not sure if what I suggested is any better :| 

```suggestion
- The Python API client for Druid. In `examples/quickstart/jupyter-notebooks/druidapi` of the Druid repo,
```

Reply:
Updated to include clone and cd instructions